### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by kenan-rhoton**
Simple guidelines for bug reports and feature requests.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (Masclins) [inline .github/ISSUE_TEMPLATE/bug_report.md]: This seems a bit useless, at least right now that we don't intend to make the game usable from smartphone

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (Masclins) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Browser's version or app's version?

**Review by Masclins — CHANGES_REQUESTED** [review]
This will be shown when creating a new issue?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (kenan-rhoton) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Figured we might as well future-proof it, but we can just remove that

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (kenan-rhoton) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Browser version :thinking:

**Review by Masclins — APPROVED** [review]

**Pull request merged**